docs/mindsdb-docs/docs/sql/create/predictor.md

```diff
@@ -28,14 +22,30 @@ Where:
 | `(SELECT [column_name, ...] FROM [table_name])` | SELECT statement for selecting the data to be used for training and validation                                                        |
 | `PREDICT [target_column]`                       | where `target_column` is the column name of the target variable.                                                                      |
 
+On execution, we get:
+
+```sql
+Query OK, 0 rows affected (x.xxx sec)
+```
+
 !!! TIP "Checking the status of the model"
-After you run the `#!sql CREATE PREDICTOR` statement, you can check the status of the training model, by selecting from the `#!sql mindsdb.predictors`
+After you run the `#!sql CREATE PREDICTOR` statement, you can check the status of the training model by selecting from the `#!sql mindsdb.predictors`.
 
-    ```sql
-    SELECT *
-    FROM mindsdb.predictors
-    WHERE name='[predictor_name]';
-    ```
+```sql
+SELECT *
+FROM mindsdb.predictors
+WHERE name='[predictor_name]';
+```
+
+On execution, we get:
+
+```sql
++------------------------+---------------------+-------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
+|name                    |status               |accuracy                 |predict                |update_status|mindsdb_version|error|select_data_query|training_options|
++------------------------+---------------------+-------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
+|predictor_name          |complete or training |[number between 0 and 1] |column_to_be_predicted |up_to_date   |22.7.5.0       |     |                 |                |
```

Comment:
- Status should be |generating or training or complete |
- Accuracy |[number between 0 and 1] | this should be changed since we have accuracy metrics that can display value larger then 1. Maybe add | number depending on the accuracy metric|

Reply:
Will be changed.

Reply:
I think on the executions we shouldn't show the options, but only one option. Open to debate this.

Reply:
@Ricram2 
Actually, in this case, there is a sample query, like this:

```
SELECT *
FROM mindsdb.predictors
WHERE name='[predictor_name]';
```

That's why all options are shown in the (sample) output table.

Do you have any suggestions to change it?